### ISSUE TITLE
bluesnarfer bt_rfcomm_config error

### ISSUE THREAD

**Issue by bluetoo1** [boos/bluesnarfer#2]
using kali linux 2018.1
command :bluesnarfer -i -r A-Z -b 68:74:4a:xx:xx:xx
device name: mymobi
bluesnarfer: open /dev/bluetooth/rfcomm/0, No such file or directory
bluesnarfer: bt_rfcomm_config failed
bluesnarfer: unable to create rfcomm connection
bluesnarfer: release rfcomm ok

Any command i use same error
also tried creating bluetooth directory /dev/bluetooth/rfcomm
but it freezes after showing device name.

**Comment by yesimxev** [comment]
```
mkdir -p /dev/bluetooth/rfcomm
mknod -m 666 /dev/bluetooth/rfcomm/0 c 216 0
```

**Comment by Rusty-Spoons13** [comment]
even after I have done this:
mkdir -p /dev/bluetooth/rfcomm
mknod -m 666 /dev/bluetooth/rfcomm/0 c 216 0

It doesn't pull any info to include device name.  Any advice?

**Comment by yesimxev** [comment]
This script is supposed to work with very old devices like old Nokia etc. For me, it doesn't work with anything

**Comment by Rusty-Spoons13** [comment]
Is there anything like this that is more up-to-date?

**Comment by Fr4nc3Sc0r** [comment]
Has a solution been found?

**Comment by yesimxev** [comment]
Not really, i think this kind of vuln works on really old bt version phones
